js/vnext/projection/column-group.js

```diff
@@ -8,13 +8,15 @@ class ColumnGroup {
     const buildColumn = ({
       name,
       width,
+      template,
       parent = null,
       columns = [],
       html = name,
       height = 1,
     }) => {
-      const col = { name, width, parent, html, height };
-
+      const col = { name, width, template, parent, html, height };
```

Comment:
All the configurations need to be included:
* headTemplate
* headHtml
* bodyTemplate
* bodyHtml
* footTemplate
* footHtml